index.mdwn

```diff
@@ -8,63 +8,64 @@ Please refer to the <a href="doc/#v4">documentation section</a> for
 news/changes and porting tips for your configuration.
 </p>
 
-# What is the awesome?
+# What is this awesome window manager?
 
 **awesome** is a highly configurable, next generation framework window
-manager for X. It is very fast, extensible and licensed under the
+manager for X. It is very fast, extendable and licensed under the
```

Comment:
Good find. Also mentioned in https://github.com/amperser/proselint/issues/604.  Will revert it.